design/features/form-metrics/version-1.md

```diff
@@ -29,100 +29,100 @@ ___
 
 #### What have we been sending  
```

Comment:
I'm a bit confused about this section as not all these bullet points refer to things we've been sending, do they? (E.g. ability to download raw data and 'Some of the Google Analytics metrics such as device type [...])

Reply:
Yeah I think this was the things Anne wanted/expected us to provide once in place - copying mural comments over was a bit difficult with this as they were muddled. Will move them to a different section. Maybe something like future expectations